apps/wallet-mobile/src/features/Discover/common/BrowserProvider.tsx

```diff
@@ -106,7 +106,7 @@ export const BrowserProvider = ({
 }
 
```

Comment:
@michaeljscript this is looking confusing and with a lot of var shadowing, can you give it a look?